iroh-net/src/portmapper.rs

```diff
@@ -415,12 +413,20 @@ impl Service {
         rx: mpsc::Receiver<Message>,
     ) -> (Self, watch::Receiver<Option<SocketAddrV4>>) {
         let (current_mapping, watcher) = CurrentMapping::new();
+        let mut full_probe = Probe::empty();
+        if let Some(in_the_past) = full_probe
+            .last_probe
+            .checked_sub(AVAILABILITY_TRUST_DURATION)
+        {
+            // we want to do a first full probe, so set is as expired on start-up
+            full_probe.last_probe = in_the_past;
```

Comment:
So this means that we won't do a probe on startup for boxes where uptime < AVAILABILITY_TRUST_DURATION.

If that is acceptable, yes this works fine.

Reply:
yes, that would be the side effect, delaying the first probe. I find it acceptable too